### PR TITLE
Add episode 17 draft

### PR DESCRIPTION
**Pull request by veggiemonk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,11 +59,13 @@ hugo new episodes/"TITLE OF SHOW.md"
 ## Podcast release process
 
 ### Preparations
+
 * Discuss the topic for the podcast in [Next Episode thread](https://devsecops.fm/episodes/next-episode/)
```
Comment (veggiemonk) [inline README.md:63]: needs updating

File: content/episodes/Best practices to build Docker images.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Notes
+
+- [Docker BuildKit](https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/experimental.md)
+- [img](https://github.com/genuinetools/img)
+- [kaniko](https://github.com/GoogleContainerTools/kaniko)
+- [buildah](https://github.com/containers/buildah)
+- [Using Secrets in build](https://docs.docker.com/develop/develop-images/build_enhancements/#new-docker-build-secret-information)
+- [Docker Slim](https://github.com/docker-slim/docker-slim)
+- [distroless](https://github.com/GoogleContainerTools/distroless)
```
Comment (veggiemonk) [inline content/episodes/Best practices to build Docker images.md:42]: to add 
```suggestion
- [distroless](https://github.com/GoogleContainerTools/distroless)
- [Docker resource comsumption](https://www.docker.com/pricing/resource-consumption-updates)
```

**Comment by Andrey9kin** [comment]
This is one should 16th not 17th or we record one more and make this one 17th

File: content/episodes/Best practices to build Docker images.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Best Practices to Build Docker Images"
+date: 2020-09-11T13:18:01+02:00
```
Comment (veggiemonk) [inline content/episodes/Best practices to build Docker images.md]:
```suggestion
date: 2020-10-12T13:18:01+02:00
```

File: content/episodes/Best practices to build Docker images.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Best Practices for Building Docker Images"
+date: 2020-10-12T13:18:01+02:00
+lastmod: 2020-10-12T13:18:01+02:00
+episode: 17
```
Comment (veggiemonk) [inline content/episodes/Best practices to build Docker images.md:5]: I'm a bit lost in the numbering.....

Comment (Andrey9kin) [reply content/episodes/Best practices to build Docker images.md:5]: I think we can drop latmod altogether. Not sure what is the purpose

**Pull request merged**